.github/workflows/export_github_data.yml

```diff
@@ -4,6 +4,11 @@ on:
   schedule:
     - cron: "20 7 * * *"
 
+permissions:
```

Comment:
Can you undo the changes to this file.  We'll update it after by making the changes in an SRE sync job.